package.json

```diff
@@ -13,25 +13,23 @@
     "npm": "3.10.10"
   },
   "scripts": {
-    "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
-    "postinstall": "node node_modules/grunt-cli/bin/grunt install",
+    "start": "node node_modules/gulp/bin/gulp build && node node_modules/gulp/bin/gulp",
```

Comment:
why do we have to run gulp twice?

Reply:
I did that because we might have to refactor our code to es6 which might give problems on heroic